apps/pipeline/upstream/env/aws/secrets-manager.yaml

```diff
@@ -13,6 +13,12 @@ spec:
       key: username
     - objectName: "pass"
       key: password
+    - objectName: "host"
```

Comment:
move secrets manager related files to a common dir?

remember to update the steps in readmes with any changes required

Reply:
Which all files are we talking here ? Does the common dir lives under aws?